tutorials/basic-topics/sorting/merge-sort.md

```diff
@@ -211,76 +239,267 @@ Call `mergeSort` to use this template to sort by divide and conquer as explained
         }
 ```
 
-Call the function `mergeSort` in public to get the Template executed `mergeSort(nums , 0 ,nums.length -1)` 
+Call the function `mergeSort` in public to get the Template executed  `mergeSort(nums , 0 ,nums.length -1)` 
 
-In all other examples we have seen only top down approach . But merge sort can be implemented in a bottom approach too 
+In all other examples we have seen only top down approach . But merge sort can be implemented in a bottom down approach too ,That will a    **iterative technique**  for sure .
 
 
 
 #### Bottom Up Approach 
 
  ![image](https://www.simplilearn.com/ice9/free_resources_article_thumb/mergesort/merge_sort-bottom-up-img1.png)
```

Comment:
don't copy images from others

Reply:
Should I remove all the images? Many of them are taken from Medium articles.

Reply:
Yes. Don't copy other's work.